### ISSUE TITLE
Delete internal subtitles after extracting them.

### ISSUE THREAD

**Issue by john626** [pannal/Sub-Zero.bundle#559]
I think this is a good enhancement because right now, you get two identical subtitle files. Two identical subtitles is a problem if the original had hearing impaired tags, because then you have two different subtitles and you can't differentiate between them in many clients.

**Comment by ukdtom** [comment]
That's a very scary terrible idea, IMHO

IMHO dangerous to mess with peoples medias, since what if something goes wrong during the remux of the media?

And at last, AFAIK, all players shows (External) after a sub, of external

**Comment by fkick** [comment]
Agreed, internal embed subtitles should not be automatically deleted. Let’s
not mess with the media files themselves please and thank you.

On Aug 31, 2018, at 1:50 PM, Tommy Mikkelsen <notifications@github.com>
wrote:

That's a very scary terrible idea, IMHO

IMHO dangerous to mess with peoples medias, since what if something goes
wrong during the remux of the media?

And at last, AFAIK, all players shows (External) after a sub, of external

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/pannal/Sub-Zero.bundle/issues/559#issuecomment-417758081>,
or mute the thread

**Comment by john626** [comment]
I was wrong about that

**Comment by john626** [comment]
>Which clients do that? Most of them differentiate exactly between embedded and external (for example SRT internal and SRT)
I was wrong about that.

Anyways, I extract my subtitles because I want to remove HI tags (which I think is the case for many, correct me if I am wrong), and then the embedded ones become misleading and useless.
